build_cmake/configs/auto.cmake

```diff
@@ -251,27 +251,31 @@ config_lib(
     "lz4 library exists."
     LIB "lz4"
     FUNC "LZ4_versionNumber"
+    HEADER "lz4.h"
 )
 
 config_lib(
     HAVE_LIBSNAPPY
     "snappy library exists."
     LIB "snappy"
     FUNC "snappy_compress"
+    HEADER "snappy.h"
 )
 
 config_lib(
     HAVE_LIBZ
     "zlib library exists."
     LIB "z"
     FUNC "zlibVersion"
+    HEADER "zlib.h"
 )
 
 config_lib(
     HAVE_LIBZSTD
     "zstd library exists."
     LIB "zstd"
     FUNC "ZSTD_versionString"
```

Comment:
On my comment about `CONFIG_LIB_FUNC`, we can remove this line + in the other `config_lib` calls